python/lsst/ts/watcher/rules/mtm2_mirror_temperature.py

```diff
@@ -0,0 +1,153 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MTM2MirrorTemperature"]
+
+import logging
+import types
+import typing
+
+import numpy as np
+import yaml
+from lsst.ts import salobj, watcher
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class MTM2MirrorTemperature(watcher.PollingRule):
+    """Monitor the mirror temperature of main telescope M2 is out of the normal
+    range.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+    log : `logging.Logger` or None, optional
+        Parent logger. (the default is None)
+    """
+
+    def __init__(
+        self, config: types.SimpleNamespace, log: logging.Logger | None = None
+    ) -> None:
+
+        remote_name = "MTM2"
+        remote_info = RemoteInfo(
+            remote_name,
+            0,
+            poll_names=["tel_temperature"],
+        )
+        super().__init__(
+            config,
+            f"MTM2MirrorTemperature.{remote_name}",
```

Comment:
MTM2 is already in the rule name. I would drop the part after the "." and just do "MTM2MirrorTemperature"

Reply:
The name is based on here:

https://ts-watcher.lsst.io/writing_rules.html#alarm-name

```
The convention for rule names is rule_class_name.remote_name_index, where rule_class_name is the class name of a rule relative to lsst.ts.watcher.rule
```

How about I rename the alarm to be `MTMirrorTemperature.MTM2`?

Reply:
Fixed in the commit: [e5e703b](https://github.com/lsst-ts/ts_watcher/pull/83/commits/e5e703b275b024b0fd34185ef4cae12882298a75). Thanks!